src/Libraries/Microsoft.Extensions.AI.Evaluation/Microsoft.Extensions.AI.Evaluation.csproj

```diff
@@ -15,6 +15,10 @@
     <MinMutationScore>0</MinMutationScore>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <InjectExperimentalAttributeOnLegacy>true</InjectExperimentalAttributeOnLegacy>
+  </PropertyGroup>
```

Comment:
What is the purpose of this property?

Reply:
This was originally needed to make the `[Experimental]` attribute available in the legacy target frameworks. However, I have removed this now that we decided not to ship the helper on which the attribute was added before.